draft.md

```diff
@@ -129,45 +129,45 @@ The native Emotiq token (EMTQ) has deflationary economics, with the token supply
 
 ## 4.6 Implementation Details
 
-Emotiq will release a technical paper (yellowpaper) with full implementation details in the next 1-2 months.
+Emotiq has released a technical paper (yellow paper) with full implementation details, available on the website.
 
 # 5. Token Generation Event
 
-Emotiq plans to raise $60M (million) through the combination of a private sale and a public token generation event (TGE). The fundraising target has been carefully selected to enable us to reach our development milestones and jumpstart the Emotiq ecosystem. Emotiq will generate one billion EMTQ tokens, distributed as follows:
+Emotiq plans to raise $39M (million) through multiple private sale rounds. The fundraising target has been carefully selected to enable us to reach our development milestones and jumpstart the Emotiq ecosystem. Emotiq will generate one billion EMTQ tokens, distributed as follows:
 * 15% sold to seed contributors (completed Dec 2017);
-* 14% sold during the private sale;
-* 36% sold during the public TGE; and
-* 35% to be held by the Emotiq team, to be used as detailed later in this section.
+* 34% sold during the private sales;
+* 37% to be held in reserve for marketing and development;
+* 10% to be held by the Emotiq team;
 
 Emotiq will follow strict KYC/AML procedures during both stages of fundraising, with funds used to, among other things:
 * Expand the the Emotiq ecosystem through educational programs;
 * Build a world-class in-house research team;
 * Explore the different implementations of smart contract languages and their usability, to accelerate blockchain adoption across the mass-market and enterprise; and
 * Provide venture funding for early projects on the Emotiq blockchain.
 
-Emotiq will aggressively pursue development, acquire the best talent, and explore new solutions. Its vision is ambitious, with equally ambitious goals.
+Emotiq will aggressively pursue development, acquire the best talent, and explore new solutions. The vision is ambitious, with equally ambitious goals.
 
-The private sale will raise $10M (million) from institutional and large private contributors and will follow a strict selection process. Single contributions will have a minimum of $500K (thousand) and contributors will be selected on the basis of the benefit they can bring to Emotiq and the ecosystem, e.g., promotion, publicity, exchange access, etc.
+The private sales will raise $37M (million) from institutional and large private contributors and will follow a strict selection process. Single contributions will have a minimum of $100K (thousand) and contributors will be selected on the basis of the benefit they can bring to Emotiq and the ecosystem, e.g., promotion, publicity, exchange access, etc.
```

Comment:
We are raising 39M and said so at the beginning. Why 37M here?

Reply:
I subtracted the seed round to get to 37M because the total 39M is stated later. What do you think?